express/features/sign-in.feature

```diff
@@ -0,0 +1,161 @@
+    When they click on the "Your account" link
+    Then they should be redirected to the "/account" page
+    And they should see the text "Your account"
+    Given that the user is on the "/change-phone-number" page
+    Then they should see the text "Change your mobile phone number"
+    Then the user submits the mobile phone number "077$$ 900000"
+    Then the error message "Enter a UK mobile phone number using numbers only" must be displayed for the "mobileNumber" field
+
+
+  Scenario: User enters an invalid number (a number that is less than 11 characters, or does not start with 07, 44, +44 or (+44))
+    And the user submits the email "registered@gds.gov.uk"
+    And the user submits the password "this-is-not-a-common-password"
+    And the user submits the security code "123123"
+    Then they should be redirected to a page with path starting with "/client-details"
+    When they click on the "Your account" link
+    Then they should be redirected to the "/account" page
+    And they should see the text "Your account"
+    Given that the user is on the "/change-phone-number" page
+    Then they should see the text "Change your mobile phone number"
+    Then the user submits the mobile phone number "08723900"
+    Then the error message "Enter a UK mobile phone number, like 07700 900000" must be displayed for the "mobileNumber" field
+
+  Scenario: User enters less than 8 characters for their new password
+    And the user submits the email "registered@gds.gov.uk"
+    And the user submits the password "this-is-not-a-common-password"
+    And the user submits the security code "123123"
+    Then they should be redirected to a page with path starting with "/client-details"
+    When they click on the "Your account" link
+    Then they should be redirected to the "/account" page
+    And they should see the text "Your account"
+    Given that the user is on the "/change-password" page
+    Then they should see the text "Add your new password"
+    When they enter "TestPa$$word" into the "currentPassword" field
+    When they enter "NewTest" into the "password" field
+    When they click the Confirm button
+    Then the error message "Your password must be 8 characters or more" must be displayed for the "password" field
+
+
+  Scenario: User selects Show for the new password field
+    And the user submits the email "registered@gds.gov.uk"
+    And the user submits the password "this-is-not-a-common-password"
+    And the user submits the security code "123123"
+    Then they should be redirected to a page with path starting with "/client-details"
+    When they click on the "Your account" link
+    Then they should be redirected to the "/account" page
+    And they should see the text "Your account"
+    Given that the user is on the "/change-password" page
+    Then they should see the text "Add your new password"
+    When they toggle "Show" link on field "password"
+    Then they see the toggle "Hide" link on field "password"
```

Comment:
Need to verify the typed password is shown on the page (not just that the link was toggled)

Reply:
I tried, It is not present in the DOM, When I inspected. So had to rely on the fact that it is toggled.